LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Refract Project
+Copyright (c) 2015 Mark W. Foster
```

Comment:
Just curious about the purpose of this. Is this because Refract Project cannot really hold copyrights? Or that you want to retain some rights? Not against it, just curious.